src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4504,59 +4504,39 @@ namespace VariableSpeedCoils {
-        Real64 RatedCapCoolSensDes;          // Autosized rated sensible cooling capacity for reporting
-        Real64 RatedCapCoolSensUser;         // Hardsized rated sensible cooling capacity for reporting
-        Real64 EvapCondPumpElecNomPowerDes;  // Autosized evaporative condenser pump power for reporting
-        Real64 EvapCondPumpElecNomPowerUser; // Hardsized evaporative condenser pump power for reporting
-        Real64 DefrostCapacityDes;           // Autosized resistive defrost heater capacity for reporting
-        Real64 DefrostCapacityUser;          // Hardsized resistive defrost heater capacity for reporting
-        bool SizingDesRunThisAirSys;         // true if a particular air system had a Sizing:System object and system sizing done
-        bool SizingDesRunThisZone;           // true if a particular zone had a Sizing:Zone object and zone sizing was done
-        Real64 HPInletAirHumRat;             // Rated inlet air humidity ratio for heat pump water heater [kgWater/kgDryAir]
-        Real64 HPWHCoolCapacity;             // estimate cooling capacity in HPWH
-
-        UpperSpeed = state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).NumOfSpeeds;
-        NormSpeed = state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).NormSpedLevel;
-        PltSizNum = 0;
-        RatedAirFlowAutoSized = false;
-        RatedWaterFlowAutoSized = false;
-        RatedCapHeatAutoSized = false;
-        IsAutoSize = false;
+        int Mode;                     // speed level
+        Real64 rhoW;                  // water density
+        Real64 SHR;                   // sensible heat transfer ratio
+        Real64 RatedAirMassFlowRate;  // rated air mass flow rate
+        Real64 CBFRated;              // bypass factor at the rated condition, considering difference in flow rates
+        Real64 RatedInletEnth;        // rated inlet air enthalpy
+        Real64 QLoadTotal1;           // placeholder for calculating SHR
+        Real64 QLoadTotal2;           // placeholder for calculating SHR
+        Real64 AirMassFlowRatio;      // air mass flow ratio
+        Real64 WaterMassFlowRatio;    // water mass flow rate
+        Real64 RatedSourceTempCool;   // rated source temperature, space cooling mode
+        std::string CurrentObjSubfix; // Object subfix type for printing
+        bool HardSizeNoDesRun;        // Indicator to hardsize withouth sizing runs
+        bool SizingDesRunThisAirSys;  // true if a particular air system had a Sizing:System object and system sizing done
+        bool SizingDesRunThisZone;    // true if a particular zone had a Sizing:Zone object and zone sizing was done
+        Real64 HPInletAirHumRat;      // Rated inlet air humidity ratio for heat pump water heater [kgWater/kgDryAir]
+        Real64 HPWHCoolCapacity;      // estimate cooling capacity in HPWH
+
+        auto &varSpeedCoils = state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum);
```

Comment:
Why is the array named `VarSpeedCoil` and a reference to a single element of the array named `varSpeedCoils`? Feels backwards.